pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -12,17 +12,43 @@ type MTOMilestone {
   # DB Fields
   id: UUID!
   name: String! # If sourced from a Common Milestone, this will resolve from the common milestone table (you can't edit this for common milestones!)
+  mtoCommonMilestoneID: UUID
+  mtoCategoryID: UUID
   facilitatedBy: MTOFacilitator
   needBy: Time
   status: MTOMilestoneStatus!
   riskIndicator: MTORiskIndicator
```

Comment:
This is marked as `NOT NULL` in the migration, but is nullable here -- which should it be?

Reply:
You are right! I think this should be riskIndicator: MTORiskIndicator!